packages/fxa-auth-server/package.json

```diff
@@ -29,7 +29,10 @@
     "paypal-processor": "CONFIG_FILES='config/secrets.json' ts-node ./scripts/paypal-processor.ts",
     "subscription-reminders": "CONFIG_FILES='config/secrets.json' ts-node ./scripts/subscription-reminders.ts",
     "write-templates": "ts-node lib/senders/emails/render-to-disk.ts",
-    "maildev-preview": "ts-node lib/senders/emails/preview-in-maidev.ts"
+    "maildev-preview": "ts-node lib/senders/emails/preview-in-maidev.ts",
+    "start-mjml": "ts-node lib/senders/emails/bin/mjml-sever.ts",
+    "storybook": "start-storybook -p 6007 --no-version-updates",
+    "storybook-preview": "npm-run-all -p start-mjml storybook"
```

Comment:
Hmm, but it's also running the mjml server, whereas the other package.json `storybook` commands _only_ run storybook - I would think maybe we could rename this to `storybook-mjml` or something and keep the `storybook` command how it is since it's technically consistent with the other ones?

Reply:
I'll go with `storybook-mjml` for now